### DISCUSSION TITLE
Wasted space in `<rh-tile>` due to arrow

### DISCUSSION THREAD

**Discussion by marionnegp** [RedHat-UX/red-hat-design-system#2086] [Ideas]
When there's no footer content next to the arrow, `<rh-tile>` shows a lot of empty space at the bottom. Is there a good way to reduce the height of each tile in this instance? @wesleymiles set up a [comparison](https://www.figma.com/proto/O4XhbignSgRz7l5hRyhDaZ/DocX---WIP?page-id=265%3A22851&node-id=9971-347801&node-type=section&viewport=-4398%2C-4812%2C0.28&t=5OjT1xwe2teipZXA-1&scaling=contain&content-scaling=fixed&starting-point-node-id=9971%3A347801) of our tile with some other cards and tiles as examples.

Potential solutions:
- reduce space for arrow when footer content slot isn't used
- reposition arrow when footer content slot isn't used
- maybe we can implement either of the above changes in compact tile only?

**Comment by bennypowers** [comment]
We are able to detect with js if footer content is present and adjust layout to suit

CC @zhawkins wrt the performance aspects. A future version of lit SSR *may* enable us to block HTML streaming in order to parse slotted children. In the mean time, we're [developing](https://github.com/patternfly/patternfly-elements/pull/2505) has-slotted hint attributes, which content authors or CMS' could use to assert the presence of slotted content to ssr processes.

**Comment by bennypowers** [comment]
This came up yesterday on a call with @diwanshi. Collapsing the whitespace when there is no footer content would help sell the use of tile, and it's included usability affordance (the arrow) to design teams.
